_posts/2016-03-16-code-smells-part-I.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: post
+name: code-smells-part-one
+title: Code Smells – Part I
+date: 2016-03-16 12:10:00 +00:00
+author: Ana Nogal
+canonical:
+    name: my personal blog
+    href: http://www.ananogal.com/blog/code-smells-part-one/
+image:
+    src: /assets/img/custom/blog/code-smells.jpg
+    attribution:
+        text: Via Flickr/Creative Commons/Brian Fitzgerald
+tags:
+- bloaters
+- code smells
+- long classes/methods
+---
+Last weekend I was at the [SoCraTes Canaries](https://twitter.com/hashtag/socracan16) and I gave my first talk ever...oh boy! How nervous was I! But now that that has passed I was wondering what I should do with all information I collected: and then I thought, maybe it's a good idea to put it all in a nice blog post...well I guess I'll do it in more than one because it's a very big topic. Essentially if you want to develop in some subtopics as I do. So let's start and see where this can lead us to.
+
+# Bloaters
+
+Bloaters can be big methods or classes, primitive obsessions, data clumps, or long parameters list.
+
+### Long Parameter List/Data Clumps
+
+The Long Parameter List is when you have a method that has more than 3 parameters. Sometimes we see that when we receive an object, and instead of passing it all we pass some of its data. In this case, the best policy is to pass the whole object. Data Clamps are a bit different: they are, in general, primitive values that start do "get together". A good example of this is a startDate and endDate... Maybe it's worth creating a DateRange.
```

Comment:
Data clump or clamp? Either the title or the paragraph has it wrong.

Do you think it would be more readable it you used back ticks for variable and class name? For example `DateRange`.